scriptmodules/emulators/dolphin.sh

```diff
@@ -38,7 +38,7 @@ function sources_dolphin() {
 function build_dolphin() {
     mkdir build
     cd build
-    cmake .. -DCMAKE_INSTALL_PREFIX="$md_inst"
+    cmake .. -DCMAKE_INSTALL_PREFIX="$md_inst" -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_FLAGS_INIT="-static"
```

Comment:
Why is the static linking required ?

Reply:
It's not _required_, but I figured this might help reduce the chance of a build failing.